tests/test_cli.py

```diff
@@ -10,6 +10,26 @@
 DATA = Path(__file__).parent / 'data'
 
 
+def test_error_annotaiton_file():
+    runner = CliRunner()
+    with TemporaryDirectory('test-file') as folder:
+        result = runner.invoke(cli, ['error-annotation', 'file',
+                                     str(DATA / 'real.asc'),
+                                     str(Path(folder, 'out.asc'))])
+        assert_equal(result.exit_code, 0, result.exception)
```

Comment:
Why not checking that the annotations were properly added to the file? Is it already tested elsewhere?